### ISSUE TITLE
The pluggin does not work at all!

### ISSUE THREAD

**Issue by wisestTu** [ozntel/oz-image-in-editor-obsidian#111]
![image](https://user-images.githubusercontent.com/40427778/177967364-c1cf7e90-fdc1-4637-bcd7-7be1ed6bb059.png)
![image](https://user-images.githubusercontent.com/40427778/177967765-a7fa1c72-a065-4ab0-b65d-50f825e47669.png)

I am sure I turn on the plugin and all the options I need, but it doesn't work:

The command "Toggle Render All" is unfindable.
![image](https://user-images.githubusercontent.com/40427778/177969185-1ae23e5a-4b71-4b97-b299-e7d063d01965.png)

The excalidraw is not viewable.
![image](https://user-images.githubusercontent.com/40427778/177970351-4000cbb7-7a05-470b-8283-040acf7ef77f.png)

Please someone help me.

**Comment by Maxlinn** [comment]
I encoutered the same problem when i updated this plugin to the latest version(2.1.3), so i manually downloaded version 2.1.2 and it worked.

**Comment by nhan000** [comment]
Dumb question. What's the difference between this plugin and the Live Preview mode?

**Comment by Maxlinn** [comment]
> Dumb question: What's the difference between this plugin and the Live Preview mode?

TLDR: If you use Live Preview mode (feel like Typora), there is no need to use this plugin.

In early versions there is no WYSIWYG mode in obsidian, Live Preview was introduced in recent releases. 
However, due to habits and plugin compability issues, some people like me still use legacy editor rather than Live Preview.
In legacy editor's edit mode, image links are just links when editing, you can't see what the image really is, so this plugin helps.
